state/txindex/kv/kv.go

```diff
@@ -588,45 +589,59 @@ func isTagKey(key []byte) bool {
 	// The check for 3 was added to allow data indexed before (w/o the event number)
 	// to be retrieved.
 	numTags := strings.Count(string(key), tagKeySeparator)
-	return numTags == 4 || numTags == 3
+	return numTags >= 3
```

Comment:
Pls update the comment `This should be always 4` to `This should be at least 4`.

Reply:
Done! :)